core/src/main/java/org/apache/accumulo/core/spi/scan/ConfigurableScanServerSelector.java

```diff
@@ -237,11 +250,13 @@ protected static class Profile {
     int busyTimeoutMultiplier;
     String maxBusyTimeout;
     String group = ScanServerSelector.DEFAULT_SCAN_SERVER_GROUP_NAME;
-    boolean enableTabletServerFallback = true;
+    String timeToWaitForScanServers = "0s";
 
     transient boolean parsed = false;
     transient long parsedMaxBusyTimeout;
 
+    transient long parsedTimeToWaitForScanServers;
```

Comment:
Could change this to a duration and compute the Duration once in parse. This avoids allocating a Duration object every time its used.

```suggestion
    transient Duration parsedTimeToWaitForScanServers;
```